src/app/views/authentication/auth-util-components/UtilComponents.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { ITheme } from '@fluentui/react'
+
+export const utilComponentStyles = (theme: ITheme) => {
```

Comment:
Though we have always called it `UtilComponent`, can we rename it to something that says exactly what it does?

Reply:
Sure. Let me rename it to ProfileButton